### PR TITLE
WIP: update classification.md

### PR DESCRIPTION
**Pull request by doutib**

### CODE REVIEW

**Comment by shamindras** [comment]
@doutib - what is the status of this?

Can it be changed to "MRG"? If it is reviewed and the status is changed. I can MRG it into the `upstream` repository

File: doc/problem/classification.md [hunk 1]
```diff
@@ -1,26 +1,45 @@
...
+
+## Additional Remarks
 
-From our limited understanding, the results of this research might have a meaningful implication on the way we treat psychological disorders. If it turns out that nature does influence these disorders, we can probably conclude that psychological disorders is not much different than physical disabilities. Otherwise, if nature has little influence over these disorders, we can try to find way to prevent these disorders from happening. 
+From our limited understanding, the results of this research might have a meaningful implication on the way we treat psychological disorders. If it turns out that nature does influence these disorders, we can probably conclude that psychological disorders is not much different than physical disabilities. Otherwise, if nature has little influence over these disorders, we can try to find way to prevent these disorders from happening.
 
-## Initial tasks 
+Initial tasks may include:
 
 1. Clean up the existing strain_classification.py: create functions and objects.
```
Comment (jarrodmillman) [inline doc/problem/classification.md]: You may want to start providing anticipated names and call signatures for the functions that need to be written.  You might perhaps also add some example use case of calling those functions as well.

File: doc/problem/classification.md [hunk 2]
```diff
@@ -1,26 +1,45 @@
...
-## Detailed summary of statistical problem
+
+## Statement of statistical problems
 
 The researchers design the experiment as follow: they have 16 different strains of mice, and each strain has 12 almost identical mice in term of genetic. If we observe that each mouse behaves very similarly to its twins, but differently from other strains of mice, we can conclude that genetic affect mouse behaviors. The researchers record the daily activities of each mouse, for example the time it spends eating, drinking, sleeping, and wondering around its habitat. 
 
-## Data Description
+## Exploratory Analysis
+
+We dispose of a labeled data set of 16 different strains of mice. Behavioral features are recorded for each mouse and each day, one example can be the time spent eating or drinking.
```
Comment (jarrodmillman) [inline doc/problem/classification.md]: I am not sure what you mean by dispose.  Also are you sure that you have "labeled data set of 16 different strains of mice?"

File: doc/problem/classification.md [hunk 3]
```diff
@@ -1,26 +1,45 @@
...
+
+ 
+ (including how and why it maps to the Statement of Problem)
+ Exploratory Analysis
+ Data Requirements Description
+ Methodology/ Approach Description
+ Testing Framework Outline
+ Additional Remarks
+ References
+
```
Comment (jcyhong) [inline doc/problem/classification.md]: These couple of lines should be removed.

File: doc/problem/classification.md [hunk 4]
```diff
@@ -1,26 +1,45 @@
...
 
-## Data Description
+## Exploratory Analysis
+
+We dispose of a labeled data set of 16 different strains of mice. Behavioral features are recorded for each mouse and each day, one example can be the time spent eating or drinking. 
 
-We dispose of a labeled data set of 16 different strains of mice. Behavioral features are recorded for each mouse and each day. One example can be the time spent eating or drinking.  
+## Data Requirement
+
+We need a set of well design behavioral features and labels for each mouse, correponding to the strain they belong.
```
Comment (jcyhong) [inline doc/problem/classification.md]: Typos: well-designed, corresponding
Comment: Would you mind elaborating what is meant by "well-designed behavioral features and labels"?

**Comment by shamindras** [comment]
@doutib, @FengshiNiu, @hduongtrong - what is the difference between this and PR #72?

They look like MRG requests for the same file

**Comment by doutib** [comment]
PR #72 is an updated version of this pull request: I addressed the issues in PR #55. I didn't find a way to update PR #55, so I created a new one.

**Comment by dylandaniels** [comment]
@doutib so do you want to close this PR?

**Comment by jarrodmillman** [comment]
@doutib , why weren't you able to update this pull request?

It is important that you learn how to use git.  So in the future when you aren't sure how something works, please ask someone.

**Comment by jarrodmillman** [comment]
I changed the status back to WIP since it looks like you haven't addressed comments from the review yet.

**Pull request closed without merging**